compiler-plugin/src/main/kotlin/arrow/meta/quotes/Parameter.kt

```diff
@@ -20,12 +20,12 @@ fun Meta.parameter(
  * A template destructuring [Scope] for a [KtParameter]
  */
 class ParameterScope(
-  override val value: KtParameter,
-  val modality: Name? = value.modalityModifierType()?.value?.let(Name::identifier),
-  val visibility: Name? = value.visibilityModifierType()?.value?.let(Name::identifier),
-  val name: Name? = value.nameAsName,
-  val `@annotationEntries`: ScopedList<KtAnnotationEntry> = ScopedList(value.annotationEntries),
-  val type: Scope<KtTypeReference> = Scope(value.typeReference),
-  val `(typeParameters)`: ScopedList<KtTypeParameter> = ScopedList(prefix = "<", value = value.typeParameters, postfix = ">"),
-  val defaultValue: KtExpression? = value.defaultValue
+  override val value: KtParameter?,
+  val modality: Name? = value?.modalityModifierType()?.value?.let(Name::identifier),
+  val visibility: Name? = value?.visibilityModifierType()?.value?.let(Name::identifier),
+  val name: Name? = value?.nameAsName,
+  val `@annotations`: ScopedList<KtAnnotationEntry> = ScopedList(value?.annotationEntries ?: listOf()),
+  val type: Scope<KtTypeReference> = Scope(value?.typeReference),
+  val `(typeParams)`: ScopedList<KtTypeParameter> = ScopedList(prefix = "<", value = value?.typeParameters ?: listOf(), postfix = ">"),
+  val defaultValue: Scope<KtExpression> = Scope(value?.defaultValue)
```

Comment:
Thanks for going back on this! Maybe we can do a separate PR for whatever naming convention cleanups there are after this one